core/src/main/java/org/infinispan/protostream/impl/JsonUtils.java

```diff
@@ -519,7 +519,7 @@ public void onTag(int fieldNumber, FieldDescriptor fieldDescriptor, Object tagVa
                case SINT64:
                case UINT64:
                case FIXED64:
-                  jsonOut.append('\"').append(tagValue).append('\"');  //todo [anistor] why do we quote number literals?
+                  jsonOut.append(tagValue);  //todo [anistor] why do we quote number literals?
```

Comment:
I believe the TODO can be removed now.  I did not write the json mapper and not sure where it is used to judge the impact of this fix. I would have fixed it when I added the todo :)